notifications.yaml

```diff
@@ -0,0 +1,220 @@
+            $ref: "#/definitions/MessageDefinition"
+      responses: 
+        202: 
+          description: "Message accepted for delivery."
+          schema: 
+            allOf: 
+              - 
+                properties: 
+                  id: 
+                    type: "string"
+                    format: "uuid"
+                  status: 
+                    $ref: "#/definitions/MessageStatus"
+                required: 
+                  - "id"
+                  - "status"
+            description: ""
+            type: "object"
+          headers: 
+            Location: 
+              type: "string"
+              description: "Location (URL) of accepted message resource.\nSend a GET request to this URL to retrieve the message status and details."
+definitions: 
+  ProblemJson: 
+    title: "Problem Type"
+    type: "object"
+    properties: 
+      type: 
+        type: "string"
+        format: "uri"
+        description: "An absolute URI that identifies the problem type.  When dereferenced,\nit SHOULD provide human-readable documentation for the problem type\n(e.g., using HTML).\n"
+        default: "about:blank"
+        example: "https://example.com/problem/constraint-violation"
+      title: 
+        type: "string"
+        description: "A short, summary of the problem type. Written in english and readable\nfor engineers (usually not suited for non technical stakeholders and\nnot localized); example: Service Unavailable\n"
+      status: 
+        type: "integer"
+        format: "int32"
+        description: "The HTTP status code generated by the origin server for this occurrence\nof the problem.\n"
+        minimum: 100
+        maximum: 600
+        exclusiveMaximum: true
+        example: 503
+      detail: 
+        type: "string"
+        description: "A human readable explanation specific to this occurrence of the\nproblem.\n"
+        example: "Connection to database timed out"
+      instance: 
+        type: "string"
+        format: "uri"
+        description: "An absolute URI that identifies the specific occurrence of the problem.\nIt may or may not yield further information if dereferenced."
+  MessageContent: 
+    title: "MessageContent"
+    type: "object"
+    properties: 
+      title: 
+        type: "string"
+        minLength: 1
+        maxLength: 255
+        description: "The title of the message. At most 255 characters."
+      body_text: 
+        type: "string"
+        x-faker: "lorem.paragraph"
+      body_html: 
+        type: "string"
+        x-faker: "lorem.paragraph"
+    required: 
+      - "title"
+  MessageDefinition: 
+    title: "MessageDefinition"
+    type: "object"
+    properties: 
+      to: 
+        description: "One fiscal code (CF).\n"
+        type: "string"
+      dry_run: 
+        type: "boolean"
+        description: "This parameter, when set to true, allows developers to test a request without actually sending a message.\n\nThe default value is false."
```

Comment:
we need to be more detailed here in the semantics, what happens? is the message created but not delivered?